scripts/qualtrics/add_intake_questions.py

```
@@ -0,0 +1,73 @@
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(levelname)s %(name)s: %(message)s",
+)
+logger = logging.getLogger(__name__)
+
+
+def main() -> None:
+    try:
+        cfg = ScriptConfig()
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)